jlh/WeatherApp/app/src/main/res/layout/item_forecast.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:orientation="horizontal"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
```

Comment:
Hahaha! :D I will change it a bit, but if i put `wrap_content` there will be no need for scrolling - it will all fit on the screen! Which kind of defeats the entire point of the `RecyclerView` etc. That's why I put `match_parent` 😝